src/pages/collections/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/png" href="/images/favicon.png" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="../../style.css">
+  <script type="module" src="../../main.js"></script>
+  <script type="module" src="../../components/collection-component/collection-component.js"></script>
+  <script type="module" src="../../components/collection-component/load-data.js"></script>
+  <title>Film Book</title>
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,700;0,900;1,100;1,300;1,700;1,900&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header-component></header-component>
+  <main>
+    <div class="movie-wrapper">
+      <collection-component
+        id="collection-1" 
+        collection-name="Best movies of 2021" 
```

Comment:
po co tutaj te atrybuty skoro i tak to w JS podmieniamy?

Reply:
bo nie mam danych wiec tymczasowe. Usunelam